### PR TITLE
added cloud hosted instructions

### PR DESCRIPTION
**Pull request by griffinhadfield**
Adding cloud hosted instructions

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -95,9 +115,23 @@ Because the JAX-RS framework handles basic API generation for JAX-RS annotations
 tree will be generated from the `inventory` service. You can use this tree as a starting point and
 augment it with annotations and code to produce a complete OpenAPI document.
 
-Now, point to the {openapi-url}[{openapi-url}^] URL to see the generated OpenAPI tree. You can also point to the
-{openapi-url}/ui[{openapi-url}/ui^] URL for a more interactive view of the APIs.
 
+ifndef::cloud-hosted[]
+Now, point to the {openapi-url}[{openapi-url}^] URL to see the generated OpenAPI tree.
```
Comment (dmuelle) [inline README.adoc:120]: Now, point to the {openapi-url}[{openapi-url}^] URL 

--->
Now, visit the {openapi-url}[{openapi-url}^] URL

File: README.adoc [hunk 2]
```diff
@@ -95,9 +115,23 @@ Because the JAX-RS framework handles basic API generation for JAX-RS annotations
 tree will be generated from the `inventory` service. You can use this tree as a starting point and
 augment it with annotations and code to produce a complete OpenAPI document.
 
-Now, point to the {openapi-url}[{openapi-url}^] URL to see the generated OpenAPI tree. You can also point to the
-{openapi-url}/ui[{openapi-url}/ui^] URL for a more interactive view of the APIs.
 
+ifndef::cloud-hosted[]
+Now, point to the {openapi-url}[{openapi-url}^] URL to see the generated OpenAPI tree. 
+You can also point to the {openapi-url}/ui[{openapi-url}/ui^] URL for a more interactive view of the APIs.
```
Comment (dmuelle) [inline README.adoc:121]: point to
--->
visit

**Review by dmuelle — COMMENTED** [review]
a few minor edits. The index.html files look good to me

**Comment by griffinhadfield** [comment]
@dmuelle Thank you for the feedback, I have addressed both of your points.

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**